modules/upgrade/pages/k-rolling-upgrade.adoc

```diff
@@ -114,7 +114,7 @@ CAUTION: Make sure to include all existing overrides, otherwise the upgrade may
 [[upgrade]]
 == Perform a rolling upgrade
 
-A rolling upgrade involves running the following procedure on each broker, one at a time:
+A rolling upgrade involves running the following procedure on each broker, one at a time. The Redpanda Helm chart and the Redpanda Operator automate this procedure.
```

Comment:
Can we link to resources on the Redpanda Operator and Redpanda Helm chart for a quick cross-reference?